new/LIT7107Divination.xml

```diff
@@ -0,0 +1,155 @@
+                   <div type="textpart" subtype="incipit" xml:lang="gez">
+                        <ab>
+                            ፩ ቀዳማዊ፡ ዐዐዐዐዐዐዐ አላዘያን፡ ዘወጽአ፡ ለሠናይ፡ ዝንቱ፡ ቤት፡ ቤተ፡ ነፍስ፡ ወሕይወት፡ ወዝንቱ፡ ትእምርት፡ ኅሩይ፡ <sic>እሳቲዊ፡</sic> ወከዋክብት፡ አልመሻታሪ፡ 
+                            <sic>ወእመዋዕል፡</sic> ሐሙስ፡ ወእግዚአብሔር፡ <sic>የአምር፡</sic> በአሰንዮ፡ ንብረት፡ ወኵሉ፡ ፍጥረት፡ ርቱዕ፡ ወብዙኅ፡ <sic>ፍሥሐ፡</sic> ወሐሤት፡ ወፍጻሜ፡ 
+                            መፍቅድ፡ <sic>ወጸሕቅ፡</sic> <choice><sic>ወአግርር፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ወአግርሮ፡</corr></choice>
+                            ፀላእት፡ <choice><sic>ወነጸሮ፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ወነጽሮ፡</corr></choice> ሕልም፡ ወአስተጋብዖ፡ ሠናያት፡ እምለፌ፡ ወእምለፌ፡
+                            ነጊደ፡ ኢየሩሳሌም፡ ወፈጻሜ፡ ኵሉ፡ <sic>ተምኔታተ<supplied reason="omitted">፡</supplied></sic>
+                        </ab>
+                    </div>
+                </div>
+                
+                <div type="textpart" xml:id="kawakebt">
+                    <label>Chapter on divination through the planets and the lunar houses, entitled <foreign xml:lang="gez">ሐሳበ፡ ከዋክብት፡  ወመናዝል፡</foreign></label>
+                    <div type="textpart" subtype="incipit" xml:lang="gez">
+                        <ab>
+                            ሐስበ፡ ከዋክብት፡ ወመናዝል፡ ለኮከብ፡ ስም፡ ወእም፡ ለአድባር፡ ስመ፡ ሀገር፡ ለሐራ<supplied reason="omitted">፡</supplied> ዘመን፡ ከመ፡ ታዕምር፡ ላዕለ፡ እኪት፡ 
+                            ወሠናይ፡ ስሞ፡ ወሞ፡ ዓመተ፡ ምሕረት፡ ወንጌላዊ፡ ፩ ወስክ፡ <choice><sic>በ ፳፷፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">በ ፳፰፡</corr></choice>
+                            ግድፍ፡ ፩ አልሐመል፡ ኮከብ፡ ዘሚያዝያ፡ መስከረም፡ ራብዕ፡ ኆኅት፡ ስሙ፡ አድናኤል፡ ፀፀፀፀ፡ እሉ፡ ፬ ከዋክብት፡ ሰበ፡ ይሰርቅ፡ በራብዕ፡ ኆኅት፡ ለሰርቀት፡ ዘታጸንን፡ 
+                            መስዓ፡ <choice><sic>ወይመጽእ፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ወይወጽእ፡</corr></choice>
+                            <sic>እምኒ<del rend="expunctuated">ሐ</del>ሃ፡</sic> ጠል፡ ወዝናም፡
+                        </ab>
+                    </div>
+                </div>
+                
+                <div type="textpart" xml:id="mogad">
+                    <label>Divinatory chapter entitled <foreign xml:lang="gez">ሐሳበ፡ ሞገድ፡</foreign>, or "Calculation of events"</label>
+                    <div type="textpart" subtype="incipit" xml:lang="gez">
+                        <ab>
+                            ስመ፡ ባሕቲቱ፡ በበ፭<supplied reason="omitted">፡</supplied> ግድፍ፡ <choice><sic>፬</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">፩</corr></choice>
+                            ንጉሥ፡ ሠናይ፡ ብዕል፡ ቀቲል፡ ፍቅ<del rend="expunctuated">ደ</del>ረ፡ ሰ<surplus>፡</surplus>ብእ፡ መደንግፅ፡ ሐሜት፡ ሀሎ፡ መልዕልተ፡ ርዕሱ፡ ጽሑፍ፡
+                            ዓቢየ፡ <sic>ወብዓለ፡</sic> ጸጋ፡ ይከውን፡ ወሥራየ፡ ይገብሩ፡ ቦቱ፡
+                        </ab>
+                    </div>
+                </div>
+                
+                <div type="textpart" xml:id="nagar2">
+                    <label>Divinatory chapter entitled <foreign xml:lang="gez">ሐሳበ፡ ፪ ነገር፡</foreign>, or "Calculation of things. Two"</label>
```

Comment:
ሐሰበ ?

Reply:
So in the catalogue, but the other textparts have ሐሳበ. Unless this is a typo by Grébaut and Tisserant, I'm inclined to give it in the following way:
`<choice><sic>ሐሰበ፡</sic><corr resp="MV">ሐሳበ፡</corr></choice>`